README_zh-CN.md

```diff
@@ -151,6 +151,30 @@ deepspeed --module --num_gpus 2 lmdeploy.pytorch.chat \
 
 ## 量化部署
 
+### Weight Only 量化
+
+```
+#   HF_model        DECODER_LAYER          LAYER_NORM
+#   internlm     InternLMDecoderLayer   InternLMRMSNorm
+#   llama 1&2     LlamaDecoderLayer      LlamaRMSNorm
+#    qwen             QWenBlock             RMSNorm
+#   baichuan        DecoderLayer            RMSNorm
+python3 -m lmdeploy.lite.apis.calibrate \
+  --model $HF_MODEL \
+  --layer_type $DECODER_LAYER \  # Decoder Layer 对应的类名
```

Comment:
Can we use model_name to determine what layer_type is?

Reply:
Good idea, I'll fix it.